contributors/guide/review-process.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Code Review Process for the Kubernetes Project
+slug: review-processes
+description: |
+  Details of the code review process, from the perspective of someone performing
+  the review.
+---
+
+Most projects in Kubernetes follow this code review process.
+
+This pages gives you a simplified description of our [full PR testing and merge workflow][pr-workflow]
+that conveniently forgets about the existence of tests, to focus solely on the roles taken
+by reviewers and by code owners.
+
+[OWNERS Files](https://www.kubernetes.dev/docs/guide/owners/) explains how `OWNERS` files work
```

Comment:
```suggestion
[OWNERS Files](owners.md) explains how `OWNERS` files work
```
Please update all links to be relative to their location here in k/community. The files need to be browsable here and will be ingested and updated as part of generating the contributor site.